gradio/blocks.py

```diff
@@ -758,20 +759,21 @@ def from_config(
         cls,
         config: dict,
         fns: list[Callable],
-        root_url: str | None = None,
+        root_url: str,
```

Comment:
Wouldn't this be a breaking change for downstream users using `from_config`? What should they put here? (Could the docstring be improved? I wouldn't understand the implications of "an external url to use as a root URL when serving files for components in the Blocks.")

Reply:
No downstream users should be using `from_config`. It's meant to be an internal method used by `gr.load()`